pandas/tests/io/json/test_normalize.py

```diff
@@ -463,6 +463,11 @@ def test_nested_flattening_consistent(self):
         # They should be the same.
         tm.assert_frame_equal(df1, df2)
 
+    def test_nonetype_record_path(self):
+        # see gh-30148
+        # should not raise TypeError
+        json_normalize([{'state': 'Texas', 'info': None}], record_path=['info'])
```

Comment:
Can you assign the result of this expression to a variable named `result` then explicitly test what this should return via `tm.assert_frame_equal(result, expected)`? Should see this in other tests

Reply:
Sure